heka/meta/heka.yml

```diff
@@ -336,7 +336,8 @@ aggregator:
           threshold: 50
           function: percent
     - status: okay
-    # A policy which is typically used for stateless clusters
+    # A policy that is typically used to derive cluster status based
+    # on the up or down status of its members.
```

Comment:
We don't have a "up" status so I find the description a bit confusing. I find it also confusing that the description just mentions "up" and "down" while the policy actually handles "critical" and "warning" as well. If we are to provide a policy that just deals with "okay" and "down" statuses then I will not try to deal with other statuses.